services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchRulesSpec.groovy

```diff
@@ -248,6 +248,42 @@ class SwitchRulesSpec extends BaseSpecification {
         ]
     }
 
+    def "Able to validate a switch in case flow is created with protected path"() {
+        given: "A switch and a flow with protected path"
+        def flow = flowHelper.randomFlow(srcSwitch, dstSwitch)
+        flow.allocateProtectedPath = true
+
+        when: "Create a flow with protected path"
+        flowHelper.addFlow(flow)
+        assert northbound.getFlowPath(flow.id).protectedPath
+
+        then: "Validate rules on the switch"
```

Comment:
`then` block description should describe what is being verified, not the action which is being taken. Actions are described in `when` blocks

Reply:
fixed